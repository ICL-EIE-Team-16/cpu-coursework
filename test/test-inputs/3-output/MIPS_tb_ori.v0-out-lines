REG v0: OUT: ffffff57
