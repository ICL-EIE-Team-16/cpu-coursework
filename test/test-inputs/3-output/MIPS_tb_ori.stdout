Loading from test-inputs/1-binary/ori.hex.txt
WARNING: src/simple_memory.v:33: $readmemh(test-inputs/1-binary/ori.hex.txt): Not enough words in the file for the requested range [0:1023].
VCD info: dumpfile waves.vcd opened for output.
num: 10000000000000000000000000000000: 
shifted num: 00000000000000000000100000000000: 
REGFile : OUT: $zero,$at,$v0,$v1,$a0,$a1,$a2,$a3,$t0,$t1,$t2,$t3,$t4,$t5,$t6,$t7,$s0,$s1,$s2,$s3,$s4,$s5,$s6,$s7,$t8,$t9,$k0,$k1,$gp,$sp,$s8,$ra
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  xxxxxxxx
address: bfc00000
Memory error: unaligned address: 00001111
address: 00001111
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  00001111
address: 00001111
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  00001111
address: bfc00004
Memory error: unaligned address: 00001111
Memory error: unaligned address: ffffff57
address: ffffff57
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  ffffff57
address: ffffff57
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  ffffff57
address: bfc00008
shift decode: 00
Memory error: unaligned address: ffffff57
address: ffffff57
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  ffffff57
shift decode: 00
address: ffffff57
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  ffffff57
address: bfc0000c
shift decode: 00
Memory error: unaligned address: ffffff57
address: ffffff57
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  ffffff57
shift decode: 00
address: ffffff57
Memory error: Address range miss, only 1024 words after BFC00000 implemented by default. Increase RANGE parameter as required. ADDR:  ffffff57
address: bfc00010
shift decode: 00
Memory error: unaligned address: ffffff57
REG v0: OUT: ffffff57
